src/components/App/App.js

```diff
@@ -13,26 +14,46 @@ class App extends Component {
 
     this.state = {
       movies: [],
+      moviesToFilter: [],
```

Comment:
Property added to hold an unadultered array of movies to refer to when re-rendered home page.